api/db/migrations/anomaly/000_create_anomaly_schema.down.sql

```diff
@@ -0,0 +1,3 @@
+DROP TABLE IF EXISTS ANOMALIES.LABELS;
+
+DROP SCHEMA IF EXISTS ANOMALIES;
```

Comment:
`anomalies` ou `anomaly` pour le schema ?
Dans le `up` on a `CREATE SCHEMA IF NOT EXISTS ANOMALY;`
Les autres schemas sont au singulier avec les noms de table au pluriel

Reply:
Yes, c'est une erreur je passe le schéma au singulier